src/sentry/integrations/vercel/client.py

```diff
@@ -82,28 +82,32 @@ def create_deploy_webhook(self):
         return response
 
     def get_env_vars(self, vercel_project_id):
-        return self.paginate(self.GET_ENV_VAR_URL % vercel_project_id, "envs")
+        return self.get(self.GET_ENV_VAR_URL % vercel_project_id)
```

Comment:
@ceorourke don't we still need the pagination if we want to be able to get the correct `envId` in order to update it?

Reply:
@MeredithAnya I upgraded to use v6 of this endpoint which returns all 100 results (it's capped at 100)